util/errors/errors.go

```diff
@@ -23,12 +23,14 @@ func IgnoreContainerNotFoundErr(err error) error {
 	return err
 }
 
+// As close to side-effect free of a check of the transient or non-transient state of an error.
+// See LogIsTransientErr().
 func IsTransientErr(err error) bool {
```

Comment:
If this is now not used outside this module you can rename it to `isTransientError` to stop it being exported.